### PR TITLE
[ZH] Add `Gameplay/Hit_object` including subpages

### PR DESCRIPTION
**Pull request by Sunnyboy971**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Hit_object/Hyperfruit/zh.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# 超级水果
```
Comment (YumeMuzi) [inline wiki/Gameplay/Hit_object/Hyperfruit/zh.md:5]: 红果

File: wiki/Gameplay/Hyperdash/zh.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# 超级加速
```
Comment (YumeMuzi) [inline wiki/Gameplay/Hyperdash/zh.md:5]: 红果跳

File: wiki/Gameplay/Hit_object/Hit_circle/zh.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - 打击物件
+---
+
+# 打击圈
+
+*打击圈 (hit circles)* 是出现于 [osu!](/wiki/Game_mode/osu!) 与 [osu!taiko](/wiki/Game_mode/osu!taiko) [谱面](/wiki/Beatmap)中带颜色的圆形[打击物件](/wiki/Hit_object)。尽管外观在两个游戏模式中不同，它们均需要玩家跟着节拍及时点击或触碰之，基于玩家的[准确度](/wiki/Gameplay/Accuracy)给予[分数](/wiki/Gameplay/Score)。比如，如果玩家点击得太早，会减少得分。及时点击打击圈会让[血条](/wiki/Client/Interface/Health_bar)略有增长，[连击](/wiki/Beatmapping/Combo)也会使血条增长。
+
+在 osu! 中，打击圈最上面有数字，以显示其在一个连击串中的位置。打击圈外围也有在其周围收缩的[缩圈](/wiki/Hit_object/Approach_circle)。一旦两者重合，玩家就应该点击或触碰打击圈来得分。
+
+在 osu!taiko 中，打击圈作为红色或蓝色物件 （Don 或 Katu）自右向左出现。当打击圈与白色[判定](/wiki/Gameplay/Judgement)圈靠近并对齐时，需要按下正确的键来得分。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Hit_object/Hit_circle/zh.md:14]: 咚或咔

File: wiki/Gameplay/Hit_object/zh.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+# 打击物件
+
+*打击物件 (hit objects)*是游玩[谱面](/wiki/Beatmap)期间，玩家可以互动的元素。每个[游戏模式](/wiki/Game_mode)都拥有一组能够改变游戏体验的独特打击物件。对于每个游戏模式打击物件的概览，详见下表：
+
+| [osu!](/wiki/Game_mode/osu!) | [osu!taiko](/wiki/Game_mode/osu!taiko) | [osu!catch](/wiki/Game_mode/osu!catch) | [osu!mania](/wiki/Game_mode/osu!mania) |
+| :-: | :-: | :-: | :-: |
+| [打击圈](/wiki/Hit_object/Hit_circle) | 打击圈 | [水果](/wiki/Hit_object/Fruit) | 音符 |
+| [滑条](/wiki/Hit_object/Slider) | 连续击鼓 | [水果串](/wiki/Hit_object/Juice_stream) | 面条 |
+| [转盘](/wiki/Hit_object/Spinner) | Denden | [香蕉](/wiki/Hit_object/Banana) | x |
```
Comment (YumeMuzi) [inline wiki/Gameplay/Hit_object/zh.md:19]: denden 是拨浪鼓

File: wiki/Gameplay/Hit_object/zh.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+# 打击物件
+
+*打击物件 (hit objects)*是游玩[谱面](/wiki/Beatmap)期间，玩家可以互动的元素。每个[游戏模式](/wiki/Game_mode)都拥有一组能够改变游戏体验的独特打击物件。对于每个游戏模式打击物件的概览，详见下表：
+
+| [osu!](/wiki/Game_mode/osu!) | [osu!taiko](/wiki/Game_mode/osu!taiko) | [osu!catch](/wiki/Game_mode/osu!catch) | [osu!mania](/wiki/Game_mode/osu!mania) |
+| :-: | :-: | :-: | :-: |
+| [打击圈](/wiki/Hit_object/Hit_circle) | 打击圈 | [水果](/wiki/Hit_object/Fruit) | 音符 |
+| [滑条](/wiki/Hit_object/Slider) | 连续击鼓 | [水果串](/wiki/Hit_object/Juice_stream) | 面条 |
+| [转盘](/wiki/Hit_object/Spinner) | 拨浪鼓 | [香蕉](/wiki/Hit_object/Banana) | x |
```
Comment (YumeMuzi) [inline wiki/Gameplay/Hit_object/zh.md:19]:
```suggestion
| [转盘](/wiki/Hit_object/Spinner) | 转盘 | [香蕉](/wiki/Hit_object/Banana) | x |
```

想了想，osu!taiko 的 denden 应该写成“这是转盘，又称为拨浪鼓”这样的描述，统一使用转盘吧

**Review by YumeMuzi — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**